### PR TITLE
Pkg executable

### PR DESCRIPTION
**Pull request by jeznag**
To distribute the app, we need to use zeit/pkg 

That library isn't compatible with ES Modules so we need to go old school require

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  setInterval(
+    () => {
+      checkUpcomingForMeetings(upcomingEvents)
+    },
+    QUICK_NAP_DURATION * 60000,
+  );
+}
+
+exports.removeEvent = removeEvent;
```
Comment (jeznag) [inline index.js:78]: It didn't work with module.exports

File: package.json [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-  "type": "module",
   "description": "",
   "main": "index.mjs",
   "scripts": {
-    "start": "node index.mjs",
-    "test": "NODE_ENV=test node index.mjs",
-	"dev": "node src/calendar-v2.mjs"
+    "start": "node index.js",
+    "test": "NODE_ENV=test node index.js",
+    "dist": "pkg index.js --targets node14-macos-x64 --no-byte-code --output dist/LateNoMore"
```
Comment (jeznag) [inline package.json:9]: used to make an executable including nodejs

File: src/notify.js [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
+        console.log('Error in VERBAL_ALERTS()', e);
       }
     }, dialog);
-  }, pauseBetweenBarksSeconds * 1000);
+  }, PAUSE_BETWEEN_BARKS_SECONDS * 1000);
 }
 
-export function askMeetingQuestions() {
-  run(meetingQuestions => {
+function askMEETING_QUESTIONS() {
```
Comment (jeznag) [inline src/notify.js:31]: oops should change this

File: src/upcoming.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+const {
+  setUpcoming,
+  removeEvent,
+  LOOK_AHEAD_MINUTES,
+} = require('../index.js');
+const { notifyUser } = require('./notify.js');
+
+async function checkUpcomingForMeetings(upcomingEvents) {
```
Comment (jeznag) [inline src/upcoming.js:8]: Had to add upcomingEvents as a parameter as require caches variable values

**Review by jeffdechambeau — APPROVED** [review]
Merging in PR.

**Pull request closed without merging**